packages/elements/src/default-theme.css

```diff
@@ -1,4 +1,82 @@
 :root {
+  /* Colors */
+  --swui-color-blue-100: #e1eefa;
```

Comment:
Nope nope nope :)

No color names in variables. And no design system in the theme.
Move these to design-system-vars.css or similar.
Rename them to `--lhds-*`, and then have proper vars in the theme. Vars with names that describe role, not look.

For example:
```
--swui-primary-action-color: var(--lhds-color-blue-600);
```